### PR TITLE
Whisper pipeline: update readme

### PR DESCRIPTION
**Pull request by as-suvorov**

### CODE REVIEW

File: src/cpp/include/openvino/genai/whisper_pipeline.hpp [hunk 1]
```diff
@@ -58,11 +61,10 @@ class OPENVINO_GENAI_EXPORTS WhisperPipeline {
...
      * @param properties optional properties
      */
-    template <typename... Properties, typename std::enable_if<ov::util::StringAny<Properties...>::value, bool>::type = true>
-    WhisperPipeline(const std::string& models_path,
-                    const std::string& device,
-                    Properties&&... properties)
-        : WhisperPipeline(models_path, device, ov::AnyMap{std::forward<Properties>(properties)...}) { }
+    template <typename... Properties,
+              typename std::enable_if<ov::util::StringAny<Properties...>::value, bool>::type = true>
+    WhisperPipeline(const std::string& models_path, const std::string& device, Properties&&... properties)
```
Comment (ilya-lavrenov) [inline src/cpp/include/openvino/genai/whisper_pipeline.hpp:66]:
```suggestion
    WhisperPipeline(const std::filesystem::path& models_path, const std::string& device, Properties&&... properties)
```

File: tests/python_tests/test_whisper_generate_api.py [hunk 2]
```diff
@@ -67,9 +67,7 @@ def read_whisper_model(params, **tokenizer_kwargs):
         model_id,
         path,
         opt_pipe,
-        ov_genai.WhisperPipeline(
-            path, 'CPU', config={'ENABLE_MMAP': False}
-        ),
+        ov_genai.WhisperPipeline(path, "CPU", config={"ENABLE_MMAP": False}),
```
Comment (ilya-lavrenov) [inline tests/python_tests/test_whisper_generate_api.py:70]: after this PR https://github.com/openvinotoolkit/openvino.genai/pull/1042 being merged, we don't have `config`

**Review by ilya-lavrenov — APPROVED** [review]

File: samples/cpp/whisper_speech_recognition/README.md [hunk 3]
```diff
@@ -17,24 +17,21 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
 
 Prepare audio file in wav format with sampling rate 16k Hz.
 
-## Run
+```sh
+wget https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/librispeech_s5/how_are_you_doing_today.wav
```
Comment (as-suvorov) [inline samples/cpp/whisper_speech_recognition/README.md:21]: @ilya-lavrenov @Wovchena Is it ok to use sample from ov models_contrib storage?

Comment (Wovchena) [reply samples/cpp/whisper_speech_recognition/README.md:21]: Yes. Why in doubts?

Comment (Wovchena) [inline samples/cpp/whisper_speech_recognition/README.md:21]: You can just post a link. It's up to a reader how to download it.

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/cpp/whisper_speech_recognition/README.md [hunk 4]
```diff
@@ -17,24 +17,21 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
...
 
-`whisper_speech_recognition whisper-base sample.wav`
+## Run
 
-Output: text transcription of `sample.wav`
+`whisper_speech_recognition whisper-base how_are_you_doing_today.wav`
 
-Models can be downloaded from [OpenAI HuggingFace](https://huggingface.co/openai).
+Output:
+```sh
```
[the remark this answers is not in the document]
Comment (as-suvorov) [reply samples/cpp/whisper_speech_recognition/README.md]: done

File: samples/cpp/whisper_speech_recognition/README.md (hunk 3 above)
Comment (as-suvorov) [reply samples/cpp/whisper_speech_recognition/README.md]: done

Comment (as-suvorov) [reply samples/cpp/whisper_speech_recognition/README.md]: Just to double check, thanks

**Review by Wovchena — APPROVED** [review]

**Comment by Wovchena** [comment]
There's a conflict

**Comment by ilya-lavrenov** [comment]
> There's a conflict

once again

**Pull request merged**